server/routes/incidents.js

```diff
@@ -151,7 +151,27 @@ module.exports = function CreateReportRoutes({ statementService, offenderService
       if (req.body.additionalComment && req.body.additionalComment.trim().length) {
         await statementService.saveAdditionalComment(statement.id, req.body.additionalComment)
       }
-      return res.redirect(`/${reportId}/your-statement`)
+      return res.redirect(`/`)
+    },
+
+    addCommentToStatement: async (req, res) => {
```

Comment:
can we call this `viewAddCommentToStatement` so it matches the same convention as the other routes?

Reply:
yes